.pre-commit-config.yaml

```diff
@@ -20,13 +20,12 @@ repos:
     hooks:
       - id: sensitive-data-leak-urls
         name: Sensitive data leak - URLs
-        entry: sbb\.ch\b
+        entry: (?<!polarion-opensource@)(?<!www\.)sbb\.ch
```

Comment:
it does not recognise:
- u123446@sbb.ch
- <XXXXX-opensourceq@sbb.ch>

Reply:
and this one? XXXX-polarion-opensource@sbb.ch

Reply:
No, but I'm not sure that makes sense. Are you sure you want to cover this case as well?

Reply:
no